modules/st2-details/style.less

```diff
@@ -272,9 +277,11 @@
 
     display: flex;
 
+    color: #848484;
```

Comment:
![2015-03-18 10 14 10](https://cloud.githubusercontent.com/assets/1357357/6702556/31e91dec-cd4f-11e4-8fd5-9ec68fe4c9db.png)

Seriously, this gray on orange looks really bad. I don't mind graying out right, inactive part, but for an active one we need to find another way to make a difference for hover... or just don't touch it at all.